ui/App.tsx

```diff
@@ -1212,7 +1256,7 @@ export const App = function() {
         <div class="u-flex u-items-center u-justify-between u-px-16 u-mt-18">
 
           <div class="c-fill-stroke-selector" ref={fillOrStrokeSelector} onClick={fillOrStrokeHandle} data-has-fill="true" data-has-stroke="true" data-active="fill" >
-            
```

Comment:
Sending PR with code style changes together with feature is not cool.

It is always better to keep the current formatting and send them in another PR.

Reply:
yeah, it was some auto-changes, fixed